themes/default/content/docs/reference/cli/environment-variables.md

```diff
@@ -155,10 +163,23 @@ meta_desc: A list of different environment variables the Pulumi CLI supports.
     </dt>
     <dd>
         <p>
-            Set this environment variable to opt-in to using `yarn` instead of `npm` for installing Node.js dependencies.
+            Set this environment variable to opt-in to using {{% md %}}`yarn`{{% /md %}} instead of {{% md %}}`npm`{{% /md %}} for installing Node.js dependencies.
         </p>
         <pre><code class="text-xs">PULUMI_PREFER_YARN=true</code></pre>
     </dd>
+    <dt>
+        <span class="font-mono">
+            PULUMI_PYTHON_CMD
+        </span>
+    </dt>
+    <dd>
+        <p>
+            As of <a href="https://github.com/pulumi/pulumi/blob/master/CHANGELOG.md#0166-2018-11-28"><code>v0.16.6</code></a>,
+            the Pulumi CLI uses {{% md %}}`python3`{{% /md %}} instead of {{% md %}}`python`{{% /md %}} when running a Python program. Set this environment variable to
```

Comment:
```suggestion
            the Pulumi CLI uses <code>python3</code> instead of <code>python</code> when running a Python program. Set this environment variable to
```